schema.graphql

```diff
@@ -92,12 +93,37 @@ type AssessmentDetail {
   id: ID!
   assessment: Assessment!
   assessmentMetrics: [AssessmentMetric!] @hasInverse(field: assessmentDetail)
-  verseTexts: [VerseText]
+  verseTexts: [VerseText!]
+  startPosition: Int
+  endPosition: Int
+  refVerseTexts: [VerseText]
+  refStartPosition: Int
+  refEndPosition: Int
+  annotations: [Annotation] @hasInverse(field: assessmentDetail)
 }
 
 type AssessmentMetric {
   id: ID!
   assessmentDetail: AssessmentDetail!
   flag: Boolean @search
   number: Float @search
+  tag: String @search
 }
+
+enum AnnotationType {
+  SubAssessment
+  VersePart
+  PartOfSpeech
```

Comment:
Are these things purely envisioned? That is, we don't really have methods that would use them now? I mean we could have PoS taggers currently for high resourced languages, but for many translations we won't have this information.

Reply:
I'm not sure if we want to add in envisioned types yet, before we are able to do something with them. It seems like a lot could change in the interim between when we envision these and then when we actually implement them. I recommend that we take out items where we don't have data yet, but that we take the envisioned feature/schema change and document it in a GitHub issue.